### PR TITLE
[Content Improvements] Update landing page layout

### PR DESCRIPTION
**Pull request by kaishin**
This PR introduces the layout we recently explored in the Getting Started page to the landing page. The stylesheets aren't production-ready yet as there is a growing amount of duplication, but we can tackle that before merging into `main`.

No media or visual elements were introduced in the PR, but it hopefully gets us closer to the goal.

[Live preview](https://swift-org-website-kaishin.vercel.app)

preview

### CODE REVIEW

**Comment by tomerd** [comment]
nice 👍

**Review by alexandersandberg — APPROVED** [review]
This is great! 🙂

**Comment by 0xTim** [comment]
These look great!

**Comment by kaishin** [comment]
@dempseyatgithub Happy to add these in this PR once we figure out some technicalities. Let's chat.

File: index.md [hunk 1]
```diff
@@ -8,17 +8,42 @@ atom: true
...
+  </ul>
 </div>
 
-## Why Swift?
+## why swift?
 
 <ul class="why-swift-list">
   <li>
-    <h3>Safe</h3>
+    <h3>safe</h3>
```
Comment (alexandersandberg) [inline index.md:46]: I think these changes were unintended.

**Comment by tomerd** [comment]
very nice 👍

**Pull request merged**